intrinsic_funcs/12_miscellaneous_vector_functions.md

```diff
@@ -568,4 +568,146 @@ vfloat64m1_t vundefined_f64m1 ();
+vuint16m2_t vset_u16m2_u16m1 (vuint16m2_t dest, size_t index, vuint16m1_t val);
+vuint16m4_t vset_u16m4_u16m1 (vuint16m4_t dest, size_t index, vuint16m1_t val);
+vuint16m4_t vset_u16m4_u16m2 (vuint16m4_t dest, size_t index, vuint16m2_t val);
+vuint16m8_t vset_u16m8_u16m1 (vuint16m8_t dest, size_t index, vuint16m1_t val);
+vuint16m8_t vset_u16m8_u16m2 (vuint16m8_t dest, size_t index, vuint16m2_t val);
+vuint16m8_t vset_u16m8_u16m4 (vuint16m8_t dest, size_t index, vuint16m4_t val);
+vuint32m2_t vset_u32m2_u32m1 (vuint32m2_t dest, size_t index, vuint32m1_t val);
+vuint32m4_t vset_u32m4_u32m1 (vuint32m4_t dest, size_t index, vuint32m1_t val);
+vuint32m4_t vset_u32m4_u32m2 (vuint32m4_t dest, size_t index, vuint32m2_t val);
+vuint32m8_t vset_u32m8_u32m1 (vuint32m8_t dest, size_t index, vuint32m1_t val);
+vuint32m8_t vset_u32m8_u32m2 (vuint32m8_t dest, size_t index, vuint32m2_t val);
+vuint32m8_t vset_u32m8_u32m4 (vuint32m8_t dest, size_t index, vuint32m4_t val);
+vuint64m2_t vset_u64m2_u64m1 (vuint64m2_t dest, size_t index, vuint64m1_t val);
+vuint64m4_t vset_u64m4_u64m1 (vuint64m4_t dest, size_t index, vuint64m1_t val);
+vuint64m4_t vset_u64m4_u64m2 (vuint64m4_t dest, size_t index, vuint64m2_t val);
+vuint64m8_t vset_u64m8_u64m1 (vuint64m8_t dest, size_t index, vuint64m1_t val);
+vuint64m8_t vset_u64m8_u64m2 (vuint64m8_t dest, size_t index, vuint64m2_t val);
+vuint64m8_t vset_u64m8_u64m4 (vuint64m8_t dest, size_t index, vuint64m4_t val);
+vfloat16m2_t vset_f16m2_f16m1 (vfloat16m2_t dest, size_t index, vfloat16m1_t val);
+vfloat16m4_t vset_f16m4_f16m1 (vfloat16m4_t dest, size_t index, vfloat16m1_t val);
+vfloat16m4_t vset_f16m4_f16m2 (vfloat16m4_t dest, size_t index, vfloat16m2_t val);
+vfloat16m8_t vset_f16m8_f16m1 (vfloat16m8_t dest, size_t index, vfloat16m1_t val);
+vfloat16m8_t vset_f16m8_f16m2 (vfloat16m8_t dest, size_t index, vfloat16m2_t val);
+vfloat16m8_t vset_f16m8_f16m4 (vfloat16m8_t dest, size_t index, vfloat16m4_t val);
+vfloat32m2_t vset_f32m2_f32m1 (vfloat32m2_t dest, size_t index, vfloat32m1_t val);
+vfloat32m4_t vset_f32m4_f32m1 (vfloat32m4_t dest, size_t index, vfloat32m1_t val);
+vfloat32m4_t vset_f32m4_f32m2 (vfloat32m4_t dest, size_t index, vfloat32m2_t val);
+vfloat32m8_t vset_f32m8_f32m1 (vfloat32m8_t dest, size_t index, vfloat32m1_t val);
+vfloat32m8_t vset_f32m8_f32m2 (vfloat32m8_t dest, size_t index, vfloat32m2_t val);
+vfloat32m8_t vset_f32m8_f32m4 (vfloat32m8_t dest, size_t index, vfloat32m4_t val);
+vfloat64m2_t vset_f64m2_f64m1 (vfloat64m2_t dest, size_t index, vfloat64m1_t val);
+vfloat64m4_t vset_f64m4_f64m1 (vfloat64m4_t dest, size_t index, vfloat64m1_t val);
+vfloat64m4_t vset_f64m4_f64m2 (vfloat64m4_t dest, size_t index, vfloat64m2_t val);
+vfloat64m8_t vset_f64m8_f64m1 (vfloat64m8_t dest, size_t index, vfloat64m1_t val);
+vfloat64m8_t vset_f64m8_f64m2 (vfloat64m8_t dest, size_t index, vfloat64m2_t val);
+vfloat64m8_t vset_f64m8_f64m4 (vfloat64m8_t dest, size_t index, vfloat64m4_t val);
```

Comment:
Naming of intrinsic must end with return type.

So
`vfloat64m8_t vset_f64m8_f64m4 (vfloat64m8_t dest, size_t index, vfloat64m4_t val);`
should be
`vfloat64m8_t vset_f64m4_f64m8 (vfloat64m8_t dest, size_t index, vfloat64m4_t val);`

See also: https://github.com/riscv/rvv-intrinsic-doc/blob/master/rvv-intrinsic-rfc.md#naming-rules

Reply:
Fixed.